recipe/meta.yaml

```diff
@@ -24,8 +24,11 @@ requirements:
     - cmake
     - make
```

Comment:
If you're cross-compiling with Python then you additionally need

```
    - python  # [build_platform != target_platform]
    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```

as build requirements.

Reply:
You've lost cmake 😄

Reply:
Yeah I typoed and then it looks like my later revision didn't make it in. 😬

Reply:
I fixed it :) do I need to trigger a rerender?

Reply:
Nope. Should be good I think.

Reply:
Great! Thanks. Now this PR probably (truly) OK and ready.